java/ingest/ingest-runner/src/main/java/sleeper/ingest/runner/task/ECSIngestTaskRunner.java

```diff
@@ -74,42 +72,37 @@ public static void main(String[] args) throws ObjectFactoryException {
         String s3Bucket = args[0];
 
         Instant startTime = Instant.now();
-        AmazonDynamoDB dynamoDBClient = buildAwsV1Client(AmazonDynamoDBClientBuilder.standard());
-        AmazonSQS sqsClient = buildAwsV1Client(AmazonSQSClientBuilder.standard());
-        AmazonCloudWatch cloudWatchClient = buildAwsV1Client(AmazonCloudWatchClientBuilder.standard());
-        AmazonS3 s3Client = buildAwsV1Client(AmazonS3ClientBuilder.standard());
 
-        try {
+        try (DynamoDbClient dynamoDBClient = buildAwsV2Client(DynamoDbClient.builder());
+                SqsClient sqsClient = buildAwsV2Client(SqsClient.builder());
+                CloudWatchClient cloudWatchClient = buildAwsV2Client(CloudWatchClient.builder());
+                S3Client s3Client = buildAwsV2Client(S3Client.builder());
+                S3AsyncClient s3AsyncClient = buildAwsV2Client(S3AsyncClient.crtBuilder());
+                S3TransferManager s3TransferManager = S3TransferManager.builder().s3Client(s3AsyncClient).build()) {
+
             InstanceProperties instanceProperties = S3InstanceProperties.loadFromBucket(s3Client, s3Bucket);
 
-            ObjectFactory objectFactory = new S3UserJarsLoader(instanceProperties, s3Client, "/tmp").buildObjectFactory();
+            ObjectFactory objectFactory = new S3UserJarsLoader(instanceProperties, s3Client, Path.of("/tmp")).buildObjectFactory();
             String localDir = "/mnt/scratch";
             String taskId = UUID.randomUUID().toString();
 
             IngestTask ingestTask = createIngestTask(objectFactory, instanceProperties, localDir,
                     taskId, s3Client, dynamoDBClient, sqsClient, cloudWatchClient,
-                    AsyncS3PartitionFileWriterFactory.s3AsyncClientFromProperties(instanceProperties),
-                    ingestHadoopConfiguration(instanceProperties));
+                    S3TransferManagerWrapper.s3AsyncClientFromProperties(instanceProperties),
```

Comment:
It looks like this was a problem before, but this client isn't closed. It should be included in the try-with-resources.

I suppose there should still be two separate S3AsyncClient objects, because the one used to upload files during ingest has specific instance properties for its configuration.